terraform/modules/helm/secor/main.tf

```diff
@@ -8,6 +8,7 @@ resource "helm_release" "secor_sa" {
   force_update     = true
   cleanup_on_fail  = true
   atomic           = true
+  depends_on       = [helm_release.secor]
```

Comment:
Why should secor_sa helm release be dependent on `helm_release.secor`? It should be the other way around.